docs/topics/resources/vert-x/vertx-starter-example-pom.xml

```diff
@@ -40,6 +40,23 @@
     </dependency>
   </dependencies>
 
+<!-- Specify the repositories containing RHOAR artifacts -->
```

Comment:
Maybe indent this line like the other lines -- I guess it's visible in the rendered docs.